libraries/security_cmd.rb

```diff
@@ -0,0 +1,44 @@
+include Chef::Mixin::ShellOut
+
+module MacOS
+  class SecurityCommand
+    attr_reader :cert
+    attr_reader :keychain
+    attr_reader :security
```

Comment:
If this attribute is the path to the command, let's indicate that with the name. Something like `:security_executable`, `:security_command` or `:security_cmd`

Reply:
done